package.json

```diff
@@ -38,8 +38,8 @@
     "graphql-iso-date": "^3.6.1",
     "graphql-resolvers": "^0.2.2",
     "jsonwebtoken": "^8.4.0",
-    "pg": "^7.6.1",
-    "sequelize": "^4.41.2",
-    "uuid": "^3.3.2"
+    "mongoose": "^5.3.14",
+    "uuid": "^3.3.2",
+    "validator": "^10.9.0"
```

Comment:
Is validator needed because Mongoose doesn't come with internal validation mechanisms?

Reply:
yes. mongoose have limited validation. for example it does not have email validation so we use validator library or write our own regex function to validate Email